ArjunaJTA/object_store/run.sh

```
@@ -37,3 +37,37 @@ mvn -e exec:java -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.InfinispanS
+
+mvn -e exec:java -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.JGroupsRaftSlotStoreConfigExample $NARAYANA_VERSION_PARAM
+if [ "$?" != "0" ]; then
+	exit -1
+fi
+
+mvn -e exec:java -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.JGroupsRaftSlotStoreExample $NARAYANA_VERSION_PARAM
+if [ "$?" != "0" ]; then
+	exit -1
+fi
+
+mvn -e exec:java -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.JGroupsRaftSlotStoreClusterExample $NARAYANA_VERSION_PARAM
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)